README.md

```diff
@@ -25,7 +25,7 @@ Then use Docker to build it.
 ```sh
 git clone https://github.com/jmhardison/docker-moodle
 cd docker-moodle
-docker build -t moodle
+docker build . -t moodle
```

Comment:
It's customary to more so put the 'context' for a docker build command at the end:

```suggestion
docker build -t moodle .
```